src/app/analytics/dashboard-widget/dashboard-widget.component.scss

```
@@ -580,102 +580,104 @@
-  .leaflet-map-element {
-    filter: invert(100%) hue-rotate(180deg) brightness(95%) contrast(90%);
-  }
-
-  .leaflet-custom-popup {
-    .leaflet-popup-content-wrapper {
-      background: rgb(30 30 30 / 85%);
-      border-color: rgb(255 255 255 / 8%);
-      box-shadow:
-        0 10px 25px -5px rgb(0 0 0 / 40%),
-        0 8px 10px -6px rgb(0 0 0 / 40%);
+  ::ng-deep {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)